pkg/proxy/ipvs/proxier.go

```diff
@@ -1027,6 +1037,8 @@ func (proxier *Proxier) syncProxyRules() {
 
 	klog.V(3).InfoS("Syncing ipvs proxier rules")
 
+	proxier.serviceNoEndpointsTotalInternal = 0
```

Comment:
Same comment here, would rename this serviceNoLocalEndpointsTotalnternal and serviceNoLocalEndpointsTotalExternal

Reply:
Done